spec/sections/010-intro.bs

```diff
@@ -0,0 +1,26 @@
+*This section is non-normative.*
```

Comment:
I'm worried about starting the files with the section indices. If we decide to add section in the middle, we'd have to rename files, which is bad for history

Reply:
History is generally pretty robust to renames. But that's why I have `010-` etc, then you can insert a section with `015-`. They're not supposed to be section numbers (though I don't know how to make that obvious).